apps/omg_watcher/lib/omg_watcher/block_getter.ex

```diff
@@ -317,8 +316,9 @@ defmodule OMG.Watcher.BlockGetter do
   end
 
   @decorate trace(tracer: OMG.Watcher.Tracer, type: :backend, service: :block_getter)
-  defp get_block_submitted_events(block_from, block_to),
-    do: EthereumEventAggregator.block_submitted(block_from, block_to)
+  defp get_block_submitted_events(block_from, block_to) do
+    RootChain.get_block_submitted_events(block_from, block_to)
```

Comment:
this is actually a partial revert from:
https://github.com/omgnetwork/elixir-omg/pull/1376/files#diff-5fcd99569a2ce28c62745940476ced06L312